### DISCUSSION TITLE
How to create a custom command to open a specific file within Nota?

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#196] [Questions] [answered]
Say I have a file called `favourites.md` that holds links to my favourite notes. It's located at `WORKSPACE/folder/favourites.md`.
How can I create a custom command with the name `favourites` that I can access via `CMD + SHIFT + P > Favourites`?

**Comment by minthemiddle** [comment, marked as answer]
- Create a custom command (`cmd + shift + p` > New custom command)
- Navigate to the custom command folder (it's in `WORKSPACE/.nota/commands`)
- Rename the script from `my-command.sh` to `favourites.sh`
- Open the script and edit as listed below
- Save
- Hit `cmd + shift + p` and type `favourites`
- Access favourites note quickly
- Alternatively, you can also set a keyboard shortcut 

favourites.md
```bash
#!/bin/bash
# @parameters workspace

cd "$1" && open "folder/favourites.md"
```

If nota is not your default Markdown editor, install the CLI tool (`cmd + shift + p` > Install CLI) and modify the script slightly:

```bash
#!/bin/bash
# @parameters workspace

cd "$1" && nota "folder/favourites.md"
```

**Reply by astoilkov** [reply]
Did you just answer your own question 😃 ? Few more suggestions:
- Nota CLI can be used to open the file (if it's not the default `.md` app) — `nota "folder/favorites.md"`. Don't forget to install it first vie the `Install global "nota" CLI...` in the Command Palette...
- You can assign a direct shortcut to the command through Preferences → Shortcuts.
- I have a similar command for my `tmp` file which is a quick way to write down things quickly.

**Reply by minthemiddle** [reply]
@astoilkov As I could not find the process described in the docs (yet), I wanted to share how I solved my own problem :) Maybe an own label (in Github's issues or discussions) for sharing scripts might encourage more people to share.

**Reply by astoilkov** [reply]
Thanks for that! We've planned better documentation for extensions. Also `notaapp/extensions` repo where we can show sample extensions.
